### ISSUE TITLE
Optional value is not recognized when typing is imported

### ISSUE THREAD

**Issue by chbndrhnns** [koxudaxi/pydantic-pycharm-plugin#257]
**Describe the bug**

If `Optional` is declared via `typing.Optional`, PyCharm complains about a missing value for the optional field.

**To Reproduce**

```py
import typing

from pydantic import BaseModel

class Response(BaseModel):
    """Container for responses sent to the client"""
    data: typing.Optional


class TestDataWrapper:
    def test_can_be_empty(self):
        assert Response().json() == '{"data": null}'

```

**Expected behavior**
It should not matter if I declare `Optional` or `typing.Optional`.

**Screenshots**
<img width="582" alt="Bildschirmfoto 2021-03-09 um 8 58 38 AM" src="https://user-images.githubusercontent.com/7534547/110437523-c35cb200-80b5-11eb-9f85-945bdbb7dfdc.png">

**Environments (please complete the following information):**
 - IDE: PyCharm Professional 2020.3
 - OS:  macOS  10.14.6
 - Pydantic Version: 1.7.3
 - Plugin version: 0.1.19

**Additional context**

**Comment by koxudaxi** [comment]
@chbndrhnns 
Thank you for creating this issue.
OK, I  will fix it.

**Comment by koxudaxi** [comment]
@chbndrhnns 
The fixed version has been released as version `0.2.0`!!
Thank you.
